master/internal/db/bun_filters.go

```diff
@@ -19,30 +20,46 @@ type FilterComparison[T any] struct {
 	Lte *T
 }
 
-func applyFieldFilterComparison[T any](
+func applyFieldFilterComparison[T any, U string | schema.Ident](
```

Comment:
I tried to make this `SelectExpr | Ident` but it doesn't _seem_ to work. It's possible the error was in where I was constructing the expression but Bun was blowing the stack when I tried to interpolate a SelectExpr.